common/djangoapps/student/models.py

```diff
@@ -59,6 +59,26 @@
 AUDIT_LOG = logging.getLogger("audit")
 SessionStore = import_module(settings.SESSION_ENGINE).SessionStore  # pylint: disable=invalid-name
 
+UNENROLLED_TO_ALLOWEDTOENROLL = _('from unenrolled to allowed to enroll')
+ALLOWEDTOENROLL_TO_ENROLLED = _('from allowed to enroll to enrolled')
+ENROLLED_TO_ENROLLED = _('from enrolled to enrolled')
+ENROLLED_TO_UNENROLLED = _('from enrolled to unenrolled')
+UNENROLLED_TO_ENROLLED = _('from unenrolled to enrolled')
+ALLOWEDTOENROLL_TO_UNENROLLED = _('from allowed to enroll to enrolled')
+UNENROLLED_TO_UNENROLLED = _('from unenrolled to unenrolled')
```

Comment:
But will they be based on the language where the server which is generating the report lives, or the instructor's language? I don't think we have translated these sort of things in the past (same sort of issue as translating log messages).